### DISCUSSION TITLE
Usage of the Enrichment extension

### DISCUSSION THREAD

**Discussion by LucChatty** [helsenorge/structor#407] [Q&A] [answered]
Hello,
While browsing the different available options, I found out about the Enrichment option.
I was wondering about what does it add compared to the definition field ?
![image](https://user-images.githubusercontent.com/89461756/144209140-69f3351f-6a44-4898-8358-95b45e01ce65.png)
for example, wouldn't "extension":[{"url":"http://ehelse.no/fhir/StructureDefinition/sdf-fhirpath","valueString":"Patient.name.where(use = 'official').family"}] simply be "definition":"http://hl7.org/fhir/StructureDefinition/Patient#Patient.name.where(use = 'official').family"  ?

Best,

Luc

**Comment by kennethmyhra** [comment, marked as answer]
Hey @LucChatty, I have not looked into `Questionnaire.item.definition`, and I am unsure if it the value after # is FhirPath or something else.

We decided to use FhirPath as per the SDC specification: http://build.fhir.org/ig/HL7/sdc/StructureDefinition-sdc-questionnaire-initialExpression.html

For some reason we created our own extension back then (STU3), probably because we could not find the `sdc-questionnaire-initialExpression`.
